src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(MAX_NUMBER + 1));
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. Here, you are creating a new instance of `ColorSupplier` and `Random` every time the `getRandomBall()` method is called, which is unnecessary. Consider creating a single instance of `ColorSupplier` and `Random` as class-level fields and reusing them in this method.